server/routes/createGoal/createGoalController.ts

```diff
@@ -20,10 +20,15 @@ export default class CreateGoalController {
 
   getCreateGoalView: RequestHandler = async (req, res, next): Promise<void> => {
     const { prisonNumber } = req.params
+    if (!req.session.createGoalForm) {
```

Comment:
I've moved the creation of the form and putting it in the session up towards the top of the GET method. It makes more sense for it to be here 👍